--- ../src/builtins.c	2022-09-26 15:24:31.421093273 -0400
+++ ckstyle/builtins.c.gnu	2022-09-26 15:24:41.037188484 -0400
@@ -17,13 +17,14 @@
 echo (char *message)
 {
   if (message != NULL)
-    message[strlen (message) - 1] = '\0'; 
+    message[strlen (message) - 1] = '\0';
   char *token = strtok (message, "\\n");
   do
     {
-      printf("%s\n", token);
+      printf ("%s\n", token);
       token = strtok (NULL, "\\n");
-    } while (token != NULL);
+    }
+  while (token != NULL);
   return 0;
 }
 
@@ -32,18 +33,14 @@
 //
 // Returns 0 on success, 1 for an invalid pair string (kvpair is NULL or
 // there is no '=' in the string).
-int
-export (char *kvpair)
-{
-  return 0;
-}
+int export(char *kvpair) { return 0; }
 
 // Prints the current working directory (see getcwd()). Returns 0.
 int
 pwd (void)
 {
   char currDirectory[1000];
-  if (getcwd(currDirectory, sizeof(currDirectory)) != NULL)
+  if (getcwd (currDirectory, sizeof (currDirectory)) != NULL)
     printf ("%s\n", currDirectory);
   return 0;
 }
@@ -72,6 +69,6 @@
   if (isBuiltIn (cmdline))
     printf ("%s: dukesh built-in command\n", cmdline);
   else
-    printf ("%s\n", cmdline); 
+    printf ("%s\n", cmdline);
   return 0;
 }
