--- ../src/builtins.c	2022-09-28 16:08:34.351534219 -0400
+++ ckstyle/builtins.c.gnu	2022-09-28 17:43:34.869029218 -0400
@@ -1,9 +1,9 @@
+#include "process.h"
+#include "shell.h"
 #include <stdio.h>
-#include <string.h>
 #include <stdlib.h>
+#include <string.h>
 #include <unistd.h>
-#include "shell.h"
-#include "process.h"
 
 // Given a message as input, print it to the screen followed by a
 // newline ('\n'). If the message contains the two-byte escape sequence
@@ -20,13 +20,14 @@
   if (message != NULL)
     {
       message[strlen (message) - 1] = '\0';
-    } 
+    }
   char *token = strtok (message, "\\n");
   do
     {
-      printf("%s\n", token);
+      printf ("%s\n", token);
       token = strtok (NULL, "\\n");
-    } while (token != NULL);
+    }
+  while (token != NULL);
   return 0;
 }
 // Given a key-value pair string (e.g., "alpha=beta"), insert the mapping
@@ -34,18 +35,14 @@
 //
 // Returns 0 on success, 1 for an invalid pair string (kvpair is NULL or
 // there is no '=' in the string).
-int
-export (char *kvpair)
-{
-  return 0;
-}
+int export(char *kvpair) { return 0; }
 
 // Prints the current working directory (see getcwd()). Returns 0.
 int
 pwd (void)
 {
   char dir[1000];
-  if (getcwd(dir, sizeof(dir)) != NULL)
+  if (getcwd (dir, sizeof (dir)) != NULL)
     printf ("%s\n", dir);
   return 0;
 }
@@ -79,19 +76,19 @@
           }
       }
   if (isBuiltIn (cmdline))
-  {
-    printf ("%s: dukesh built-in command\n", cmdline); 
-    return 0;  
-  }
+    {
+      printf ("%s: dukesh built-in command\n", cmdline);
+      return 0;
+    }
   else if (strncmp (cmdline, "./", 2) == 0)
-  {
-    if (isExecutable (&cmdline[3]))
-      printf ("%s: is executable\n", cmdline);
-    printf ("%s\n", cmdline);
-    return 0;
-  }
+    {
+      if (isExecutable (&cmdline[3]))
+        printf ("%s: is executable\n", cmdline);
+      printf ("%s\n", cmdline);
+      return 0;
+    }
   getcwd (PATH, sizeof (PATH));
-  printf("%s/%s\n", PATH, cmdline);
+  printf ("%s/%s\n", PATH, cmdline);
   return 0;
 }
 // Changes the current working directory.
@@ -101,7 +98,7 @@
 {
   if (newDir == NULL)
     return 1;
-  newDir [strlen (newDir) - 1] = '\0';
+  newDir[strlen (newDir) - 1] = '\0';
   if (chdir (newDir) != 0)
     perror ("cd failed");
   strcpy (PATH, newDir);
