--- ../src/builtins.c	2022-09-25 18:33:56.778201510 -0400
+++ ckstyle/builtins.c.gnu	2022-09-25 18:34:08.334317325 -0400
@@ -15,13 +15,14 @@
 echo (char *message)
 {
   if (message != NULL)
-    message[strlen (message) - 1] = '\0'; 
+    message[strlen (message) - 1] = '\0';
   char *token = strtok (message, "\\n");
   do
     {
-      printf("%s\n", token);
+      printf ("%s\n", token);
       token = strtok (NULL, "\\n");
-    } while (token != NULL);
+    }
+  while (token != NULL);
   return 0;
 }
 
@@ -30,18 +31,14 @@
 //
 // Returns 0 on success, 1 for an invalid pair string (kvpair is NULL or
 // there is no '=' in the string).
-int
-export (char *kvpair)
-{
-  return 0;
-}
+int export(char *kvpair) { return 0; }
 
 // Prints the current working directory (see getcwd()). Returns 0.
 int
 pwd (void)
 {
   char currDirectory[1000];
-  if (getcwd(currDirectory, sizeof(currDirectory)) != NULL)
+  if (getcwd (currDirectory, sizeof (currDirectory)) != NULL)
     printf ("%s\n", currDirectory);
   return 0;
 }
