--- ../src/builtins.c	2022-09-25 18:09:13.063332003 -0400
+++ ckstyle/builtins.c.gnu	2022-09-25 18:10:59.676400406 -0400
@@ -16,13 +16,14 @@
   if (message != NULL)
     {
       message[strlen (message) - 1] = '\0';
-    } 
+    }
   char *token = strtok (message, "\\n");
   do
     {
-      printf("%s\n", token);
+      printf ("%s\n", token);
       token = strtok (NULL, "\\n");
-    } while (token != NULL);
+    }
+  while (token != NULL);
   return 0;
 }
 
@@ -31,11 +32,7 @@
 //
 // Returns 0 on success, 1 for an invalid pair string (kvpair is NULL or
 // there is no '=' in the string).
-int
-export (char *kvpair)
-{
-  return 0;
-}
+int export(char *kvpair) { return 0; }
 
 // Prints the current working directory (see getcwd()). Returns 0.
 int
