--- ../src/process.c	2022-09-26 15:18:16.309380161 -0400
+++ ckstyle/process.c.gnu	2022-09-26 15:33:18.010308245 -0400
@@ -15,8 +15,8 @@
   for (int i = 0; i < sizeof (builtInCommands) / sizeof (builtInCommands[0]);
        i++)
     {
-      if (strncmp (command, builtInCommands[i],
-                   strlen (builtInCommands[i])) == 0)
+      if (strncmp (command, builtInCommands[i], strlen (builtInCommands[i]))
+          == 0)
         return 1;
     }
   return 0;
