--- ../src/process.c	2022-09-29 10:35:34.993264872 -0400
+++ ckstyle/process.c.gnu	2022-09-29 10:40:35.260043330 -0400
@@ -60,11 +60,10 @@
       dup2 (fd[1], STDOUT_FILENO);
       if (strlen (writeableArgument) == 0)
         execlp (command, command, NULL);
+      else if (token2 == NULL || token1 == NULL)
+        execlp (command, command, writeableArgument, NULL);
       else
-        if (token2 == NULL || token1 == NULL)
-          execlp (command, command, writeableArgument, NULL);
-        else
-          execlp (command, command, token1, token2, NULL);
+        execlp (command, command, token1, token2, NULL);
     }
   char buffer[1000];
   for (int i = 0; i < sizeof (buffer); i++)
