--- ../src/shell.c	2022-09-28 16:51:51.891928410 -0400
+++ ckstyle/shell.c.gnu	2022-09-28 17:43:35.061030997 -0400
@@ -31,7 +31,7 @@
       if (fgets (buffer, MAXLENGTH, input) == NULL)
         {
           break;
-        } 
+        }
       if (input != stdin)
         printf ("%s", buffer);
       if (strncmp (buffer, "echo", 4) == 0)
@@ -48,7 +48,7 @@
           runCmd ("./bin/ls", arguments);
         }
       if (strncmp (buffer, "quit", 4) == 0)
-        break; 
+        break;
     }
   printf ("\n");
   hash_destroy ();
