--- ../src/shell.c	2022-09-28 22:31:47.862736751 -0400
+++ ckstyle/shell.c.gnu	2022-09-29 10:40:35.352044181 -0400
@@ -31,7 +31,7 @@
       if (fgets (buffer, MAXLENGTH, input) == NULL)
         {
           break;
-        } 
+        }
       if (input != stdin)
         printf ("%s", buffer);
       if (strncmp (buffer, "echo", 4) == 0)
@@ -55,10 +55,10 @@
       if (strncmp (buffer, "export", 6) == 0)
         {
           char *arguments = &buffer[7];
-          export (arguments); 
+          export(arguments);
         }
       if (strncmp (buffer, "quit", 4) == 0)
-        break; 
+        break;
     }
   printf ("\n");
   hash_destroy ();
